public/src/test/java/com/rackspace/salus/telemetry/api/web/services/TestMonitorAndEventTaskServiceTest.java

```diff
@@ -0,0 +1,113 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.rackspace.salus.telemetry.api.web.services;
+
+import static com.rackspace.salus.common.util.SpringResourceUtils.readContent;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.Assert.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.doReturn;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.rackspace.salus.event.manage.model.TestTaskResult;
+import com.rackspace.salus.event.manage.model.TestTaskResult.EventResult;
+import com.rackspace.salus.event.manage.model.kapacitor.KapacitorEvent.EventData;
+import com.rackspace.salus.event.manage.model.kapacitor.KapacitorEvent.SeriesItem;
+import com.rackspace.salus.event.manage.model.kapacitor.Task.Stats;
+import com.rackspace.salus.event.manage.web.client.EventTaskApi;
+import com.rackspace.salus.monitor_management.web.client.MonitorApi;
+import com.rackspace.salus.monitor_management.web.model.TestMonitorInput;
+import com.rackspace.salus.monitor_management.web.model.TestMonitorOutput;
+import com.rackspace.salus.telemetry.api.model.TestMonitorAndEventTaskRequest;
+import com.rackspace.salus.telemetry.api.model.TestMonitorAndEventTaskResponse;
+import com.rackspace.salus.telemetry.api.services.TestMonitorAndEventTaskService;
+import com.rackspace.salus.telemetry.model.SimpleNameTagValueMetric;
+import java.util.List;
+import java.util.Map;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.co.jemos.podam.api.PodamFactory;
+import uk.co.jemos.podam.api.PodamFactoryImpl;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class TestMonitorAndEventTaskServiceTest {
+
+  private PodamFactory podamFactory = new PodamFactoryImpl();
+
+  @Autowired
+  private TestMonitorAndEventTaskService testMonitorAndEventTaskService;
+
+  @Autowired
+  ObjectMapper objectMapper;
+
+  @MockBean
+  MonitorApi monitorApi;
+
+  @MockBean
+  EventTaskApi eventTaskApi;
+
+  @Test
+  public void testGetTestMonitorAndEventTask() throws Exception {
+    TestMonitorAndEventTaskRequest testMonitorAndEventTaskRequest = objectMapper
+        .readValue(readContent("testCreateTestMonitorAndEventTask_req.json"),
+            TestMonitorAndEventTaskRequest.class);
+    String tenantId = "aaaaaa";
```

Comment:
Unless hardcoding the value is required it is good practice to use this instead.
```suggestion
    String tenantId = RandomStringUtils.randomAlphabetic(8);
```